src/main/resources/image_wikimedia_commons.bpp

```diff
@@ -14,7 +14,7 @@ $(upload.description)
 |other_versions=
 }}
 # } else {
-$(upload.description)	
+$(upload.description.startsWith("{{Information") ? upload.description : common.description)
```

Comment:
i'm not sure this makes sense (or made sense before...) shouldn't we have _both_ descriptions regardless of whether we have an information template or not?

Reply:
That was to avoid having two {{Information}} template, but allowing both would allow more advanced formatting (e.g. defining the Author in common but Description in upload).